FlowCrypt/Functionality/Services/EnterpriseServerApi.swift

```diff
@@ -71,9 +71,9 @@ class EnterpriseServerApi: EnterpriseServerApiType {
         .recoverFromTimeOut(result: nil)
     }
 
-    func getDomainRules(for email: String) -> Promise<DomainRules?> {
-        Promise<DomainRules?> { resolve, _ in
-            guard let userDomain = email.recipientDomain,
+    func getClientConfiguration(for email: String) -> Promise<ClientConfiguration?> {
+        Promise<ClientConfiguration?> { resolve, _ in
+            guard let userDomain = "name@flowcrypt.com".recipientDomain,
```

Comment:
Before marking a PR as ready for review, please do a self-review first on github. I do this for all code I push to GitHub before I ask others to review. This way you ensure that it is really ready for review and that bugs or temporary code was not left in the PR. Thanks!

Reply:
You're, I should've been more careful. Sorry about that. I'll fix that shortly!